.circleci/config.yml

```diff
@@ -19,6 +19,25 @@ jobs:
           root: .
           paths: [ . ]
 
+  compile-jdk11:
+    docker: [{ image: 'circleci/openjdk:11-node' }]
+    resource_class: medium+
+    steps:
+      - checkout
+      - restore_cache: { key: 'gradle-wrapper-v2-jdk11-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}' }
+      - restore_cache: { key: 'gradle-cache-v2-jdk11-{{ checksum "versions.props" }}-{{ checksum "build.gradle" }}' }
+      - run: ./gradlew --profile --parallel --stacktrace resolveConfigurations
+      - save_cache:
+          key: 'gradle-wrapper-v2-jdk11-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}'
+          paths: [ ~/.gradle/wrapper ]
+      - save_cache:
+          key: 'gradle-cache-v2-jdk11-{{ checksum "versions.props" }}-{{ checksum "build.gradle" }}'
+          paths: [ ~/.gradle/caches ]
+      - run: ./gradlew --profile --parallel --stacktrace classes testClasses
+      - persist_to_workspace:
+          root: .
+          paths: [ . ]
```

Comment:
@carterkozak this is causing the duplicate workspace files, we should just delete this.

Reply:
I'd also be in favor of either not doing check-11 (we don't do that anywhere else!), or merging it into with compile-11, such that this branch of jobs doesn't persist extra stuff to the workspace, nor require compiling with java 11 twice (if we don't persist, both `compile-11` and `check-11` would have to compile from scratch)